### PR TITLE
www-servers/caddy: add 2.7.5, update live & cleanup

### PR DESCRIPTION
**Pull request by rahilarious**
1. split out LICENSE
2. Install Caddyfile as upstream intended.
3. show correct version on `caddy version`
4. update initd confd files pointing to new config file

### CODE REVIEW

File: www-servers/caddy/caddy-2.7.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+src_compile() {
+	# https://github.com/caddyserver/caddy/blob/master/caddy.go#L843
+	if [[ ${PV} == 9999* ]]; then
+		local CUSTOM_VER="git-$(git rev-parse --short HEAD)"
+	else
+		local CUSTOM_VER="${PV}"
+	fi
+
+	ego build -ldflags "-X github.com/caddyserver/caddy/v2.CustomVersion=${CUSTOM_VER}" ./cmd/caddy
+	for sh in bash fish zsh; do
```
Comment (thesamesam) [inline www-servers/caddy/caddy-2.7.5.ebuild:69]:
```suggestion
	local sh
	for sh in bash fish zsh; do
```

File: www-servers/caddy/files/remove-binary-altering-commands-2.7.5.patch [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Remove commands which can modify binary outside of Portage.
```
Comment (thesamesam) [inline www-servers/caddy/files/remove-binary-altering-commands-2.7.5.patch:1]: Good commentary!

File: www-servers/caddy/caddy-2.7.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+		go-module_live_vendor
+	else
+		go-module_src_unpack
+	fi
+}
+
+src_prepare(){
+	default
+	sed -i -e "s|User=caddy|User=http|g;" ../dist-"${PV}"/init/*service || die
```
Comment (thesamesam) [inline www-servers/caddy/caddy-2.7.5.ebuild:56]: You could combine this into one sed by adding two `-e`s

File: www-servers/caddy/caddy-2.7.5.ebuild (hunk 1 above)
Comment (rahilarious) [reply www-servers/caddy/caddy-2.7.5.ebuild:69]: done

File: www-servers/caddy/caddy-2.7.5.ebuild (hunk 3 above)
Comment (rahilarious) [reply www-servers/caddy/caddy-2.7.5.ebuild]: done

**Comment by rahilarious** [comment]
@zmedico waiting for you

**Comment by rahilarious** [comment]
ping @zmedico

**Comment by zmedico** [comment]
Thanks!

**Pull request closed without merging**